src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -60,6 +61,16 @@ export const ExposureNotificationServiceProvider = ({
     const onAppStateChange = async (newState: AppStateStatus) => {
```

Comment:
Curious - why did you decide to fire this whenever the app state changes? won't this result in scheduling the same job multiple times? Or does the old job get cleared when the new one is scheduled?

Reply:
The use of `ExistingPeriodicWorkPolicy.REPLACE` when calling `workManager.enqueueUniquePeriodicWork` in the Kotlin code will clear out any pending jobs and replace it with the new one.